.github/workflows/deploy.yml

```diff
@@ -0,0 +1,51 @@
+name: Deploy
+on:
+  push:
+    branches: [
+      "develop",
```

Comment:
Should we really trigger this workflow on push to `develop`?

Reply:
Naah idk
We shouldnt, i guess, yeah